src/ImageSequencer.js

```diff
@@ -292,6 +299,30 @@ ImageSequencer = function ImageSequencer(options) {
     });
   }
 
+  function loadNewModule(name,options){
+
+    if(!options) {
+      return this;
+    }
+    else if(Array.isArray(options)) {
+      // contains the array of module and info
+      this.module[name] = options;
+
+    } else  if(options.func && options.info) {
+      // passed in options object
+      this.modules[name] = [
+        options.func, options.info
+      ]
+    } else if(options.path&&!this.inBrowser) {
+      // load from path
```

Comment:
Note what works in Node only?

Reply:
Actually this loads a module from a given path(basically importing it) and hence it is not available in browser. Major application for this is adding a test module